lnmarkets/umbrel-app.yml

```diff
@@ -28,6 +28,22 @@ gallery:
   - 2.jpg
   - 3.jpg
 path: ""
-deterministicPassword: true
+deterministicPassword: false
```

Comment:
Although you may have dropped the password requirement from this version of LNMarkets, can we keep this as `true` due to a bug in Umbrel atm where this will hide the password for folks with LNMarkets currently installed. I would do this myself, but unfortunately, I don't have write permissions.

Reply:
We removed a lot of logic due to that feature being optional now, and since we had a breaking change (the reason why we update in the first place) current version does not work currently so the update is pretty much forced at this point.

Reply:
@kibotrel You can enable this no worries, it as nothing do to with the auth proxy. It will basically just display the deterministic password in the App Store UI for LNM, that our app no longer use. It's just a temporary fix.

Reply:
Yes exactly; nothing to do with App Proxy here; will just continue to display the password in the UI even though it's not needed and will address a temporary problem. The next Umbrel update addresses this.

Reply:
Sure then, i'll turn it back on!